scripts/servers.lua

```diff
@@ -0,0 +1,827 @@
+return {
+  {
+    '1C Enterprise',
+    '[Contributors](https://github.com/1c-syntax/bsl-language-server/graphs/contributors)',
+    '[BSL Language Server](https://github.com/1c-syntax/bsl-language-server)',
+    'Java',
```

Comment:
if we actually want this info, it should live in the configs. Do not introduce a new, parallel source of truth.

Reply:
Yeah I fully agree, that's why I said I'm not sure how and if this table should live in the repo. I added it here to show a "parsable" version the list of this https://github.com/Microsoft/language-server-protocol/blob/gh-pages/_implementors/servers.md, and to give some suggestions about maybe creating an "update" script that can validate these URLs regularly.

Furthermore, there are about 100 servers that are currently supported by lspconfig, I would imagine going through these 100 files and updating the entries by hand to be not only tedious, but also error prone. But maybe reviewing it will take more effort 🤷‍♂️ 

I actually wanted to have the markdown table parsed from directly from the URL but I spent way too much time trying to figure out a reasonable way to do it before I gave up because it takes like 10s to fix it with a macro instead of setting up a bunch of libraries to do it. The rabbit holes can start with either https://github.com/jgm/lunamark or https://github.com/msva/lua-htmlparser if anyone would like to explore.